### PR TITLE
feat(query): added "App Service Without Latest PHP Version" query for Terraform Azure 

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: Php Version Not Latest When Running Web App

Notes: 
- Query severity and category need review.
- Commenting after a value in the samples results in the whole line being ignored (when using it in the search key).

`application_stack{

      php_version = "7.4"

    } `
The php_version line gets parsed.

`application_stack{

      php_version = "7.4" # this is a comment

    } `
The php_version line is ignored.

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	resource := input.document[i].resource.azurerm_windows_web_app[name]
+    php_version := resource.site_config.application_stack.php_version
+	php_version != "v7.4"
+    
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_windows_web_app[%s].site_config.application_stack.php_version", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "for the attribute 'php_version' to be the lastest version (7.4)",
+		"keyActualValue": "'php_version' is not the lastest version (7.4)",
```
Comment (rafaela-soares) [inline assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego:32]:
```suggestion
		"keyExpectedValue": "for the attribute 'php_version' to be the latest available stable version (8.0)",
		"keyActualValue": "'php_version' is not the latest available stable version (8.0)",
```

File: assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	resource := input.document[i].resource.azurerm_linux_web_app[name]
+    php_version := resource.site_config.application_stack.php_version
+	to_number(php_version) != 8.0
+    
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_linux_web_app[%s].site_config.application_stack.php_version", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "for the attribute 'php_version' to be the lastest version (8.0)",
+		"keyActualValue": "'php_version' is not the lastest version (8.0)",
```
Comment (rafaela-soares) [inline assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego:48]:
```suggestion
		"keyExpectedValue": "for the attribute 'php_version' to be the latest available stable version (8.0)",
		"keyActualValue": "'php_version' is not the latest available stable version (8.0)",
```

File: assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego [hunk 3]
```diff
@@ -0,0 +1,51 @@
+package Cx
+
+import data.generic.common as common_lib
+
+# for deprecated version (before AzureRM 3.0)
+CxPolicy[result] {
+	resource := input.document[i].resource.azurerm_app_service[name]
+	php_version := resource.site_config.php_version
+    to_number(php_version) != 7.4
```
Comment (rafaela-soares) [inline assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego:9]: As mentioned in the CIS Microsoft Azure Foundations Benchmark v1.4.0 (item 9.6), the PHP version should be the latest available stable release that is PHP 8.0

```suggestion
    to_number(php_version) != 8.0
```

File: assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+		"keyActualValue": "'php_version' is not the lastest version (7.4)",
+		"searchLine": common_lib.build_search_line(["resource", "azurerm_app_service", name, "site_config", "php_version"], []),
+	}
+}
+
+# After 3.0, for windows
+CxPolicy[result] {
+	resource := input.document[i].resource.azurerm_windows_web_app[name]
+    php_version := resource.site_config.application_stack.php_version
+	php_version != "v7.4"
```
Comment (rafaela-soares) [inline assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego:25]: As mentioned in the CIS Microsoft Azure Foundations Benchmark v1.4.0 (item 9.6), the PHP version should be the latest available stable release that is PHP 8.0

```suggestion
    php_version != "v8.0"
```

File: assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+	resource := input.document[i].resource.azurerm_app_service[name]
+	php_version := resource.site_config.php_version
+    to_number(php_version) != 7.4
+    
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_app_service[%s].site_config.php_version", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "for the attribute 'php_version' to be the lastest version (7.4)",
+		"keyActualValue": "'php_version' is not the lastest version (7.4)",
```
Comment (rafaela-soares) [inline assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego:16]:
```suggestion
		"keyExpectedValue": "for the attribute 'php_version' to be the latest available stable version (8.0)",
		"keyActualValue": "'php_version' is not the latest available stable version (8.0)",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego (hunk 4 above)
Comment (kaplanlior) [reply assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego:25]: Actually that 8.1 (:
https://www.php.net/downloads

Comment (rafaela-soares) [reply assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego]: Thank you, @kaplanlior! 🚀

**Review by rafaela-soares — DISMISSED** [review]
LGTM 🚀

File: assets/queries/terraform/azure/app_service_without_latest_php_version/metadata.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "96fe318e-d631-4156-99fa-9080d57280ae",
+  "queryName": "App Service Without Latest PHP Version",
+  "severity": "LOW",
+  "category": "Best Practices",
```
Comment (rjegoncalves) [inline assets/queries/terraform/azure/app_service_without_latest_php_version/metadata.json:5]: Severity and Category LGTM.

**Review by joaorufi — APPROVED** [review]

**Pull request merged**